components/renku_data_services/storage_schemas/api.spec.yaml

```diff
@@ -256,7 +256,6 @@ components:
         readonly:
           type: boolean
           description: Whether this storage should be mounted readonly or not
-          default: true
```

Comment:
Why change? The current API in the notebooks makes all storage read-only by default. We should probably respect that.

Reply:
because this is a PATCH requests, so readonly might not be passed, in which case it should be None and not modified.

Reply:
also notebooks doesn't make storage readonly anymore, it's mounted ro or rw depending on users preferences. See https://github.com/SwissDataScienceCenter/renku-notebooks/pull/1614